the-blue-alliance-ios/View Controllers/Base/Data Controllers/TBATableViewController.swift

```diff
@@ -28,8 +28,7 @@ class TBATableViewController: UITableViewController, DataController {
         tableView.delegate = self
         tableView.register(UITableViewCell.self, forCellReuseIdentifier: basicCellReuseIdentifier)
 
-        refreshControl = UIRefreshControl()
-        refreshControl?.addTarget(self, action: #selector(refresh), for: .valueChanged)
+        enableRefreshing()
```

Comment:
Don't be lazy - this pattern can go in the collection view controller as well (since it's in 2/3 already)